locales/en.json

```diff
@@ -39,19 +39,18 @@
   "https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/account-individuals.html?utm_campaign=not-applicable&utm_medium=vanity-url&utm_source=canada-ca_my-cra-account": "https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/account-individuals.html?utm_campaign=not-applicable&utm_medium=vanity-url&utm_source=canada-ca_my-cra-account",
   "canada.ca/taxes-help": "canada.ca/taxes-help",
   "canada.ca/my-cra-account": "canada.ca/my-cra-account",
-  "service.name": "Claim Tax Benefits",
+  "service.name": "Claim benefits by filing taxes",
   "start.banner.1": "The Canada Revenue Agency (CRA) sent you a letter about the new",
   "start.banner.2": " service. ",
   "start.banner.3": "You need information on that letter to file your 2018 taxes with this service.",
```

Comment:
Also we can delete these now that there's no banner. (in the french version as well)